test/ptp/tgm/validation/tests/validation.go

```diff
@@ -0,0 +1,68 @@
+package tests
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/openshift/ptp-operator/test/pkg"
+	"github.com/redhat-eets/sno-tests/test/pkg/client"
+	"github.com/redhat-eets/sno-tests/test/pkg/consts"
+	"github.com/redhat-eets/sno-tests/test/pkg/devices"
+	"github.com/redhat-eets/sno-tests/test/pkg/pods"
+)
+
+var _ = Describe("PTP T-GM Validation", func() {
+	var testPort string
+	testPort, ok := os.LookupEnv("TGM_TESTING_PORT")
```

Comment:
What about using a yaml file so that the user can detail the environment specific configuration on a declarative way? See an example below from telco QE:

https://github.com/redhat-partner-solutions/cnf-gotests/blob/vran_enablement/config/config.yaml

Reply:
Makes sense to me too. I was proposing you the yaml file because more variables are gonna be needed as we progress in the test development.